src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Main {
+    private static final int ARRAY_SIZE = 6;
+    private static final FigureSupplier figureSupplier = new FigureSupplier();
```

Comment:
Avoid using static fields for mutable objects. Here, 'figureSupplier' is a static field but it is an instance of a mutable class 'FigureSupplier'. Instead, create an instance of 'FigureSupplier' inside the 'main' method.

Reply:
Agree with comment